tests_e2e/tests/ext_sequencing/ext_sequencing.py

```diff
@@ -95,10 +96,8 @@ def _get_sorted_extension_names(extensions: List[VirtualMachineScaleSetVMExtensi
         for ext in extensions:
             # Only check extensions which succeeded provisioning
             if "succeeded" in ext.statuses_summary[0].code:
-                enabled_time = ssh_client.run_command(f"ext_sequencing-get_ext_enable_time.py --ext '{extension_full_names[ext.name]}'", use_sudo=True)
-                formatted_time = datetime.strptime(enabled_time.strip(), u'%Y-%m-%dT%H:%M:%SZ')
-                if formatted_time < test_case_start:
```

Comment:
This check is no longer needed. ext_sequencing-get_ext_enable_time.py now only checks for logs after this timestamp